deployment/cdk/opensearch-service-migration/lib/service-stacks/reindex-from-snapshot-stack.ts

```diff
@@ -106,6 +107,7 @@ export class ReindexFromSnapshotStack extends MigrationServiceCore {
                 "RFS_TARGET_USER": targetUser,
                 "RFS_TARGET_PASSWORD": targetPassword,
                 "RFS_TARGET_PASSWORD_ARN": targetPasswordArn,
+                "TUPLE_DIR_PATH": `/shared-replayer-output/traffic-replayer-${deployId}`
```

Comment:
do you mean `traffic-replayer` here?

Reply:
I'd rather place all the logs in once place even if its a little strange.  We can circle back on this and update the documentation after we have a cohesive strategy.